src/index.js

```diff
@@ -64,13 +64,18 @@ export default class TimeAgo extends Component<DefaultProps, Props, void> {
   timeoutId: ?number;
   isStillMounted: boolean = false;
 
+  // Use Number.isNaN if available, otherwise use polyfill (not available in some browsers inc. IE11)
+  checkIsNaN = (value) => {
+    return (Number.isNaN !== undefined) ? Number.isNaN(value) : typeof value === 'number' && isNaN(value);
+  }
```

Comment:
This function works by checking that Number.isNaN is available or not (IE up to and including 11 do not support it), and then returning appropriately. 

I agree that this should probably be abstracted into another utility function, and I will try and do so when I get the chance. Maybe I will even use another NPM module, such as https://www.npmjs.com/package/number.isnan.

@nmn - There is a difference between Number.isNaN and isNaN, and whilst I have added a polyfill already, I was asked to contribute this - so I have. I would suggest that you at least support all browser versions that React currently does (>IE8), as otherwise this is not a useful repo for users in real-world applications. Babel also does not automatically add the Number.isNaN polyfill, either.